clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -2798,6 +2813,15 @@ void cir::FuncOp::print(OpAsmPrinter &p) {
     p.printAttribute(annotations);
   }
 
+  if (getCxxSpecialMember()) {
+    p << " cxx_special_member<";
```

Comment:
Why do we need to wrap it in `cxx_special_member`? It is apparent from `cir.cxx_ctor/dtor` that this is the special member function.